src/utils/events/move-input.js

```diff
@@ -25,12 +25,14 @@ export default class MoveInput {
     this.options.events.forEach(event => this.element.removeEventListener(event, this.handleEvent));
   }
 
-  set(options) {
-    Object.assign(this.options, options);
-  }
-
-  isSourceOf(eventName) {
-    return MOVE_EVENT_TYPES.indexOf(eventName) >= 0;
+  /**
+   * Enable this input (begin processing events)
+   * if the specified event type is among those handled by this input.
+   */
+  enableOnEventType(eventType) {
```

Comment:
Should we drop the `On` and just call this `enableEventType`? Are we saying something important with the `On`?

Do we only need the ability to enable event types (not disable them)?